### DISCUSSION TITLE
recent ui update broke reload ui and apply and restart button too

### DISCUSSION THREAD

**Discussion by NaotoEnjoyer** [TheLastBen/fast-stable-diffusion#1439] [Q&A]
as title say, after another ui update i cant apply newly added extensions when i click either button gradio tries to reload but its 1. breaks and says there are no gradio running atm or after longer time host new adress and when i open it ui is bugged everything loading and i get error box that some event is closed
also noticed that live previews doesnt work too.. damn what a nice update auto.. and localtunnel also has same issues as well as gradio

**Comment by TheLastBen** [comment]
are you using the latest colab from the link, sometimes an outdated notebook can cause some issues with the ui

**Reply by NaotoEnjoyer** [reply]
yes i believe i do use updated colab i literally open same link each time https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb#scrollTo=p4wj_txjP3TC
and it is updating seeing new changes from time to time, recent webui ui update broke reload ui and restart and apply functions

**Reply by TheLastBen** [reply]
how to reproduce the error ?

**Reply by NaotoEnjoyer** [reply]
i literally said how, click on reload ui in settings or apply and restart in extensions so ui will hard restart and create new link usually but rn it fucking breaks and cant let me apply extensions i added also live previews doesnt work too

**Reply by NaotoEnjoyer** [reply]
so an update i tried and this is what happes, i click on either of those buttons, og link errors out then on refresh give me this - No interface is running right now
and after quite a long time cmd opens new link which is stuck on loading i mean webui opens but all stuff models settings etc just stuck on loading status forever.. thats just bruh

**Reply by NaotoEnjoyer** [reply]
and this is the msg i get on new link webui - Event loop is closed

**Reply by NaotoEnjoyer** [reply]
> how to reproduce the error ?

hey do you know why it takes so long for automatic to fix it? maybe i should just do clean sd install again? :/ man its becoming annoying that i have to move remove everything literally each time a in between few updates
